SPECS/meson/meson.spec

```diff
@@ -1,8 +1,8 @@
 %{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
```

Comment:
Nit: this can be removed, we define %python3_sitelib in the [mariner-rpm-macros package](https://github.com/microsoft/CBL-Mariner/blob/9402f6c19e7c90d14f1bd7b7ee1fc005feb0c7e9/SPECS/mariner-rpm-macros/macros.python3#L1).